wp-content/plugins/hurumap/src/IndicatorBlock/Save.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+
+import propTypes from '../propTypes';
+
+function Save({ attributes }) {
+  return (
+    <div
+      data-id={`indicator-block-${attributes.id}`}
+      data-title={attributes.title}
+      data-description={attributes.description}
+      data-source-title={attributes.sourceTitle}
+      data-source-link={attributes.sourceLink}
+      data-widget={attributes.widget}
+      data-src={attributes.src}
```

Comment:
These attribubutes should come from `hurumap-ui/cms` dataProps function.

eg.:

```
  save: ({ attributes }) => {
    return <div {...dataProps(TYPES.HURUMAP_CARD, attributes)} />;
  }
```

Reply:
@karimkawambwa I will be changing this once, the hurumap-ui is ready